test/native/jest.config.js

```diff
@@ -11,12 +14,12 @@ module.exports = {
     },
     transform: {
         "binding.js$": ['ts-jest'],
+        "^.+\\.ts?$": ['ts-jest'],
     },
     testPathIgnorePatterns: [
         '/node_modules/'
     ],
     modulePathIgnorePatterns: [
         '/cmake-js/'
     ],
-    testRunner: "jest-circus/runner"
 }
```

Comment:
trivial: lots of these files have no newline at end of file.

Reply:
The JS linter actually removes them